projectEuler/problem7.js

```diff
@@ -0,0 +1,31 @@
+// prompt: find the 10001st prime number
+
+// importing isPrime function
+function isPrime(wholeNum) {
+  var count = 0;
```

Comment:
Note: Always use `let` or `const` instead of `var` in JavaScript.

Here is my `isPrime` function. I may have shared this with you in the past but it is the most performant solution I've found and have re-used it many times:

````
function isPrime(n) {
  const maxFactor = Math.sqrt(n);
  
  // Handle the special case of receiving 2
  if (n === 2) {
    return true;
  }
  
  // Handle negatives, 0, 1, and even numbers
  if (n < 2 || n % 2 === 0) {
    return false;
  }
  
  for (let i = 3; i <= maxFactor; i += 2) {    
    if(n % i === 0) {
      return false;
    }
  }
  
  return true;
}
````